frontend/javascripts/admin/dataset/dataset_upload_view.tsx

```diff
@@ -490,16 +490,16 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
 
       if (fileExtension === "zip") {
         try {
-          const reader = new ZipReader(new BlobReader(file));
+          const reader = new Zip.ZipReader(new Zip.BlobReader(file));
           const entries = await reader.getEntries();
           await reader.close();
-          const wkwFile = entries.find((entry: Entry) =>
+          const wkwFile = entries.find((entry: any) =>
```

Comment:
I know this isn't pretty, but Zip.Entry doesnt work because the import of the wrapper is somehow not found

Reply:
I just pushed a commit (I hope the CI passes), which improves the typing a bit.

Reply:
nice, thank you!